src/hotspot/share/runtime/continuation.hpp

```diff
@@ -47,9 +47,26 @@ void continuations_init();
 class javaVFrame;
 class JavaThread;
 
+// should match Continuation.toPreemptStatus() in Continuation.java
```

Comment:
I can't find Continuation.toPreemptStatus() and the enum in Continuation.java doesn't match.

Reply:
Should be just PreemptStatus. Fixed.